Makefile

```diff
@@ -1,27 +1,3 @@
-ARCH ?= amd64
-OS ?= $(shell uname -s | tr '[:upper:]' '[:lower:'])
-CURL ?= curl --fail -sSL
-XARGS ?= xargs -I {}
-
-BIN_DIR ?= ${HOME}/bin
-PATH := $(BIN_DIR):$(PATH)
-
-VERSION ?= $$(grep -E '^current_version' .bumpversion.cfg | sed 's/^.*= //')
```

Comment:
Looks like this and the `cfn/version` target need to be added back...

Reply:
Okay